packages/keystone/executionContext.js

```diff
@@ -57,5 +57,6 @@ function _internalGetAsyncLocalStorage (name = 'default') {
 
 module.exports = {
     _internalGetExecutionContextAsyncLocalStorage,
+    _internalGetAsyncLocalStorage,
```

Comment:
why is the `_internalGetExecutionContextAsyncLocalStorage` is not enought?

Reply:
Since execution context in my head = request context, it contains `reqId` and other stuff.

`gqlContext` -> GQL call stack, if request contains multiple queries = its multiple contexts, if mutation calls another mutation / query -> that's a call stack (right now not implemented, since not required)...

So I guess its just different context from definition...

Reply:
@toplenboren could you please check

Reply:
Because @SavelevMatthew wants custom context, not the one that we use for reqId storage

Why couldnt we just squeeze context into `executionContext`? It would also help in traces and logs. I mean, just add one more param besides `reqId` `execId` `taskId`... like `graphQLOperation`